python/python/glide/async_commands/core.py

```diff
@@ -2262,3 +2262,32 @@ async def invoke_script(
                 ["foo", "bar"]
         """
         return await self._execute_script(script.get_hash(), keys, args)
+
+    async def pfadd(self, key: str, elements: List[str]) -> int:
+        """
+        Adds all elements to the HyperLogLog data structure stored at the specified `key`.
+        Creates a new structure if the `key` does not exist.
+
+        When no `elements` are provided, and `key` exists and is a HyperLogLog, then no operation is performed.
```

Comment:
should that mean that elements are optional? or just passing an empty list?

Reply:
I think pptional is for commands which have multiple cmd formats, like `set`: `SET key value [NX | XX] [GET] ...`